docs/getting-started/roadmap.md

```diff
@@ -4,27 +4,35 @@ This document defines a high level roadmap for KubeEdge development.
 
 The [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) represent the most up-to-date plans.
 
-KubeEdge 1.2 is our current stable branch. The roadmap below outlines new features that will be added to KubeEdge.
+KubeEdge 1.3 is our current stable branch. The roadmap below outlines new features that will be added to KubeEdge.
 
-## 2020 Q1 Roadmap
-- Support for monitoring at the edge.
-- Support HA for cloudcore
-- Support exec&logs API for edge application
-- Add protobuf support for data exchange format between cloud and edge
-- Finish scalability test and publish report
-- Support managing clusters at edge from cloud (aka. EdgeSite)
-- Enhance performance and reliability of KubeEdge infrastructure.
-- Support ingress at edge.
-- Improve contributor experience by defining project governance policies, release process, membership rules etc.
-- Improve the performance and e2e tests with more metrics and scenarios.
+## 2020 Q2 Roadmap
+
+- Support metrics-server in the cloud.
+- Support Kubernetes exec API for edge application.
+- Upgrade Kubernetes dependency to 1.18.
+- Support edgenode certificate rotation.
+- Upgrade golang to 1.14.
+- Support ingress/gateway at edge.
+- Device CRD improvement, support device protocol extension.
+- Edge nodes cross subnet communication.
+- Support list-watch from edgecore for applications on the edge.
+- Collect data information sent from the edge side from CloudHub
 - Improve KubeEdge installation experience
 - Add more docs and move docs out of main repo
 
+
 ## Future
+
+- Improve contributor experience by defining project governance policies, release process, membership rules etc.
+- Improve the performance and e2e tests with more metrics and scenarios.
```

Comment:
As discussed with fei xu,  arm e2e might be included in this bullet,  let me do some investigation first.

Reply:
OK, I think add `Add integration and e2e tests on ARM in CI` in future list?

Reply:
+1 will talk with you on this later.